src/Plugins/LevelDBStore/IO/Data/LevelDB/DB.cs

```diff
@@ -126,18 +126,14 @@ public void Write(WriteOptions options, WriteBatch write_batch)
             NativeHelper.CheckError(error);
         }
 
-        public IEnumerable<KeyValuePair<byte[], byte[]>> GetAll(Snapshot? snapshot = null)
+        public IEnumerator<KeyValuePair<byte[], byte[]>> GetEnumerator()
```

Comment:
I think we don't need this enumerator in the database, Snapshot is enough

Reply:
This is different than snapshot. Two different things.